### PR TITLE
[13.0][MIG] privacy

### PR DESCRIPTION
**Pull request by techultrasolutions**
Data Privacy and Protection
======================

This application provides data protection and privacy feature,
to comply with regulations such as GDPR.

### CODE REVIEW

**Comment by techultrasolutions** [comment]
@pedrobaeza @dreispt 
Would you please review this PR?

**Comment by techultrasolutions** [comment]
> @pedrobaeza @dreispt
> Would you please review this PR?

@pedrobaeza ping

**Comment by pedrobaeza** [comment]
Sorry, I don't know enough the module for functional review, but in a first glance, you haven't followed the migration guide:

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

that states that you have to do the pre-commit stuff in one first commit.

Please change that and I'll do code review later, and my colleague @Yajo may review it functionally.

**Review by dreispt — APPROVED** [review]

**Comment by techultrasolutions** [comment]
> Sorry, I don't know enough the module for functional review, but in a first glance, you haven't followed the migration guide:
> 
> https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0
> 
> that states that you have to do the pre-commit stuff in one first commit.
> 
> Please change that and I'll do code review later, and my colleague @Yajo may review it functionally.

@pedrobaeza Above mentioned points are done.

**Comment by techultrasolutions** [comment]
@max3903 Please Review.

File: privacy/demo/res_users.xml [hunk 1]
```diff
@@ -1,8 +1,7 @@
-<?xml version="1.0" encoding="utf-8" ?>
 <!-- Copyright 2018 Tecnativa - Jairo Llopis
      License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
-<data>
+<odoo>
     <record id="base.user_demo" model="res.users">
         <field name="groups_id" eval="[(4, ref('group_data_protection_user'))]" />
     </record>
-</data>
+</odoo>
```
Comment (yajo) [inline privacy/demo/res_users.xml:7]: No need to add this change, it was like that on purpose.

File: privacy/security/data_protection.xml [hunk 2]
```diff
@@ -1,31 +1,25 @@
-<?xml version="1.0" encoding="utf-8" ?>
 <!-- Copyright 2018 Eficent Business and IT Consulting Services S.L.
      License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
 <odoo>
-    <data noupdate="0">
```
Comment (yajo) [inline privacy/security/data_protection.xml]: This data should remain as noupdate=0 AFAIK.

File: privacy/views/data_protection_menu_view.xml [hunk 3]
```diff
@@ -59,4 +57,4 @@
         groups="group_data_protection_manager"
         sequence="90"
     />
-</data>
+</odoo>
```
Comment (yajo) [inline privacy/views/data_protection_menu_view.xml:60]: Unnecessary change. Undo please.

File: privacy/views/privacy_activity_view.xml [hunk 4]
```diff
@@ -133,4 +132,4 @@
         id="menu_privacy_activity"
         parent="menu_data_protection_master_data"
     />
-</data>
+</odoo>
```
Comment (yajo) [inline privacy/views/privacy_activity_view.xml:135]: Undo also please.

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by yajo** [comment]
This doesn't look good. It's the activities form:

imagen

File: privacy/views/data_protection_menu_view.xml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+        sequence="30"
+    />
+    <record id="action_data_protection_config" model="ir.actions.act_window">
+        <field name="name">Data Protection Settings</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">res.config.settings</field>
+        <field name="view_mode">form</field>
+        <field name="target">inline</field>
+        <field name="context">{'module': 'data_protection'}</field>
+    </record>
```
Comment (yajo) [inline privacy/views/data_protection_menu_view.xml:51]: This menu item leads to settings, yes, but there are no privacy-related settings. Is that normal? Maybe it's a submodule that needs this hook here? Otherwise it could be removed.

File: privacy/security/data_protection.xml (hunk 2 above)
Comment (dreispt) [reply privacy/security/data_protection.xml]: @Yajo, that's the default already, so not really needed.

**Review by dreispt — APPROVED** [review]

**Review by dreispt — APPROVED** [review]
There are a couple of unneeded changes, but non blocking. Approved.

**Comment by yajo** [comment]
I was pushing an alternate PR and accidentally I pushed https://github.com/OCA/data-protection/commit/16b58ea2a6a591b1fb2051bbff8e5960fe6272c5 to the main 13.0 branch 🙈. I was testing odoo-module-migrator and got bit by https://github.com/OCA/odoo-module-migrator/pull/47, which I hope is merged soon.

I'm very sorry for that, but it's done already, so I'll close this PR. If there's anything to change, please open a new one.
